superset-frontend/plugins/plugin-chart-echarts/src/Bubble/transformProps.ts

```diff
@@ -0,0 +1,188 @@
+        ${yAxisLabel}: ${params.data[1]} <br/>
+        ${sizeLabel}: ${params.data[2]}`;
+}
+
+export default function transformProps(chartProps: EchartsBubbleChartProps) {
+  const { height, width, hooks, queriesData, formData, inContextMenu } =
+    chartProps;
+
+  const { data = [] } = queriesData[0];
+  const {
+    x,
+    y,
+    size,
+    entity,
+    maxBubbleSize,
+    colorScheme,
+    series: bubbleSeries,
+    xAxisLabel: bubbleXAxisTitle,
+    yAxisLabel: bubbleYAxisTitle,
+    xAxisFormat,
+    yAxisFormat,
+    yAxisBounds,
+    logXAxis,
+    logYAxis,
+    xAxisTitleMargin,
+    yAxisTitleMargin,
+    truncateYAxis,
+    xAxisLabelRotation,
+    yAxisLabelRotation,
+  }: EchartsBubbleFormData = { ...DEFAULT_FORM_DATA, ...formData };
+
+  const colorFn = CategoricalColorNamespace.getScale(colorScheme as string);
+
+  const legends: string[] = [];
+  const series: ScatterSeriesOption[] = [];
+
+  const xAxisLabel: string = x.label || x;
+  const yAxisLabel: string = y.label || y;
+  const sizeLabel: string = size.label || size;
+
+  data.forEach(datum => {
+    const name = (bubbleSeries ? datum[bubbleSeries] : datum[entity]) as string;
+    const bubbleSeriesValue = bubbleSeries ? datum[bubbleSeries] : null;
+
+    series.push({
+      name,
+      data: [
+        [
+          datum[xAxisLabel],
+          datum[yAxisLabel],
+          datum[size.label],
+          datum[entity],
+          bubbleSeriesValue as any,
+        ],
+      ],
+      type: 'scatter',
+      itemStyle: { color: colorFn(name) },
+    });
+    legends.push(name);
+  });
+
+  normalizeSymbolSize(series, maxBubbleSize);
+
+  const xAxisFormatter = getNumberFormatter(xAxisFormat);
+  const yAxisFormatter = getNumberFormatter(yAxisFormat);
+
+  const [min, max] = yAxisBounds.map(parseYAxisBound);
+
+  const echartOptions: EChartsCoreOption = {
+    series,
+    xAxis: {
+      axisLabel: { formatter: xAxisFormatter },
+      splitLine: {
+        lineStyle: {
+          type: 'dashed',
+        },
+      },
+      nameRotate: xAxisLabelRotation,
+      scale: true,
+      name: bubbleXAxisTitle,
+      nameLocation: 'middle',
+      nameTextStyle: {
+        fontWight: 'bolder',
+      },
+      nameGap: xAxisTitleMargin || 30,
+      type: logXAxis ? AxisType.log : AxisType.value,
+    },
+    yAxis: {
+      axisLabel: { formatter: yAxisFormatter },
+      splitLine: {
+        lineStyle: {
+          type: 'dashed',
+        },
+      },
+      nameRotate: yAxisLabelRotation,
+      scale: truncateYAxis,
+      name: bubbleYAxisTitle,
+      nameLocation: 'middle',
+      nameTextStyle: {
+        fontWight: 'bolder',
+      },
+      nameGap: yAxisTitleMargin || 50,
+      min,
+      max,
+      type: logYAxis ? AxisType.log : AxisType.value,
+    },
+    legend: {
+      ...getLegendProps(LegendType.Scroll, LegendOrientation.Top, true),
+      data: legends,
+    },
+    tooltip: {
+      show: !inContextMenu,
+      ...defaultTooltip,
```

Comment:
`defaultTooltip` seems to have been a leftover from a recent fix (my bad; I should have removed it in #22218). Please use `getDefaultTooltip` from `utils/tooltip.ts` (this ensures proper tooltip positioning) and remove `defaultTooltip` in `defaults.ts`:
```suggestion
      ...getDefaultTooltip(refs),
```